### DISCUSSION TITLE
Help with logitech-options livecheck

### DISCUSSION THREAD

**Discussion by cesarcoatl** [Homebrew/discussions#2223] [Writing Formulae/Casks] [answered]
Logitech has changed the `url` for downloading the Options installer ([homebrew-cask-drivers/Casks/logitech-options.rb](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/Casks/logitech-options.rb)). For that reason, the current `livecheck` retrieves the second to last version, i.e. `8.54.147`.

Current `livecheck`:

```ruby
  livecheck do
    url "https://support.logi.com/api/v2/help_center/en-us/articles.json?label_names=webcontent=productdownload,websoftware=ec86eb2b-8e0b-11e9-a62b-a944e73f7596"
    regex(%r{/Options[._-]?v?(\d+(?:\.\d+)+)\.zip}i)
  end
```

But the `url` for the latest version has changed.

From: <https://download01.logi.com/web/ftp/pub/techsupport/options/Options_8.54.147.zip>
To: <https://download01.logi.com/web/ftp/pub/techsupport/options/options_installer.zip>

So I'm trying to update the cask but I'm looking for assistance from the community to properly extract the version number from the `json`.

The `url` for the `livecheck` works, but the `regex` is the one that must be updated.

Rather than on the download link, now the version should be matched from the following section:

```json
<li>\n<b><span>Software Version: </span></b>9.40.75</li>
```

**Comment by miccal** [comment, marked as answer]
I have made some changes in https://github.com/Homebrew/homebrew-cask-drivers/pull/2409
